### ISSUE TITLE
Universal Steam Links Bug

### ISSUE THREAD

**Issue by w108x** [JosefNemec/PlayniteExtensions#193]
I'm using the latest playnite version 9.18 with Universal Steam Metadata and everytime I scrape metadata the link labels come out like this:

```
<!LOCSteamLinksCommunityHub!>
<!LOCSteamLinksDiscussions!>
<!LOCSteamLinksGuides!>
```

where it should be:

```
Community Hub
Discussions
Guides
```

**Comment by JosefNemec** [comment]
Fixed in 2.6 update.

**Comment by w108x** [comment]
this was the fastest bugfix in the history of bugfixes, works like intended now, thanks a ton.
